lang/python/python-yaml/Makefile

```diff
@@ -8,36 +8,62 @@
 include $(TOPDIR)/rules.mk
 
 PKG_NAME:=PyYAML
-PKG_VERSION:=3.11
+PKG_VERSION:=3.12
 PKG_RELEASE:=1
 
 PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.gz
 PKG_SOURCE_URL:=http://pyyaml.org/download/pyyaml/
-PKG_MD5SUM:=f50e08ef0fe55178479d3a618efe21db
+PKG_MD5SUM:=4c129761b661d181ebf7ff4eb2d79950
```

Comment:
note: as far as i understand it, the MD5SUM can be discarded, since the PKG_HASH field will be preferred
no harm in leaving it 

i started removing them ; 
since these packages are intended to be used with OpenWrt/LEDE trunk, they should both handle the PKG_HASH field